apps/web/src/app/globals.css

```
@@ -1,6 +1,92 @@
+    }
+    to {
+      height: 0;
+    }
+  }
+}
+
+@utility container {
+  margin-inline: auto;
+  padding-inline: 2rem;
+  @media (width >= theme(--breakpoint-md)) {
+    max-width: none;
```

Biome (1.9.4)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)